src/library_sigs.js

```diff
@@ -298,6 +298,7 @@ sigs = {
   _embind_register_value_object__sig: 'vpppppp',
   _embind_register_value_object_field__sig: 'vpppppppppp',
   _embind_register_void__sig: 'vpp',
+  _emscripten_create_wasm_worker__sig: 'ipi',
```

Comment:
Can library_wasm_worker.js define these for itself?

Reply:
The point of this file is that its auto-generated.. the individual library files no longer need `__sig` entries (in the general case).

Reply:
Oh wait, this then reads like a system JS library only machinery? How would user JS libraries generate these sigs?

Reply:
Yes, this only works for system JS libraries today.  Right now it works like `gen_struct_info.py` which is also only supported for internal/system JS libraries.

User JS functions still need to explicitly add `__sig` attributes if they want good wam64 support.

We can looking into extending this mechanism to support user-JS code at some point in the future.